controller/controller.go

```diff
@@ -95,7 +95,16 @@ func Filter(gvk schema.GroupVersionKind) func(obj interface{}) bool {
 // FilterGroupVersionKind makes it simple to create FilterFunc's for use with
 // cache.FilteringResourceEventHandler that filter based on the
 // schema.GroupVersionKind of the controlling resources.
+//
+// Deprecated: Use FilterControlledByGVK instead.
 func FilterGroupVersionKind(gvk schema.GroupVersionKind) func(obj interface{}) bool {
+	return FilterControlledByGVK(gvk)
+}
+
+// FilterControlledByGVK makes it simple to create FilterFunc's for use with
+// cache.FilteringResourceEventHandler that filter based on the
+// schema.GroupVersionKind of the controlling resources.
+func FilterControlledByGVK(gvk schema.GroupVersionKind) func(obj interface{}) bool {
```

Comment:
```suggestion
func FilterControllerGVK(gvk schema.GroupVersionKind) func(obj interface{}) bool {
```

Thoughts on this slight change?

Reply:
wfm!

Reply:
Pushed.